src/lib/tzcode/timelocal.h

```diff
@@ -0,0 +1,65 @@
+/*-
```

Comment:
From where it originates? Are we really need an SPDX identifier and a license text both?

Reply:
This time around it was from [FreeBSD lib/libc/stdtime/timelocal.h](https://github.com/freebsd/freebsd-src/blob/main/lib/libc/stdtime/timelocal.h), but simplified a bit, similarly to `stftime.h` we have got from NetBSD in step.1. But I believe they both were originating from Olson drop ages ago.